README.md

```diff
@@ -121,22 +120,13 @@ Right now everything runs in one process, so if you set a break point in your co
 One trick is to kill the `AudioContext` right before the break point, like this:
 
 ```javascript
-context._kill()
+context[Symbol.dispose]()
```

Comment:
It seems to be part of [standard proposal](https://github.com/tc39/proposal-explicit-resource-management#using-const-with-existing-resources).
The intuition here is: we need this method to be exposed since we use it from outside (therefore it is not private nor `_` prefixed which indicates internal stuff), same time not polluting "official" namespace.
It can be done via _FinalizationRegistry_ (probably should add at some point), but for now it's explicit.

See [symbol-dispose](https://github.com/spectjs/symbol-dispose).

Reply:
Aha... didn't know about this proposal. Thanks for the information 🙏